anaconda.spec.in

```diff
@@ -443,9 +449,11 @@ rm -rf \
 # do not provide the live subpackage on RHEL
 
 %files live
+%{_unitdir}/locale1-x11-sync.service
```

Comment:
The service file needs to be installed as user service file.

https://docs.fedoraproject.org/en-US/packaging-guidelines/Systemd/#packaging_filesystem

We need to store it in `/usr/lib/systemd/user` instead of the current path. The current path broke the script as it depends on user session variables. In other words use `_userunitdir` rpm macro instead.

Reply:
OK, converted to a user unit. :)